src/Microsoft.ML.Data/DataView/RowToRowMapperTransform.cs

```diff
@@ -88,7 +88,12 @@ private static VersionInfo GetVersionInfo()
 
         public override DataViewSchema OutputSchema => _bindings.Schema;
 
-        bool ICanSaveOnnx.CanSaveOnnx(OnnxContext ctx) => _mapper is ICanSaveOnnx onnxMapper ? onnxMapper.CanSaveOnnx(ctx) : false;
+        bool ICanSaveOnnx.CanSaveOnnx(OnnxContext ctx)
+        {
+            const int minimumOpSetVersion = 9;
+            Contracts.Assert(ctx.GetOpSetVersion() >= minimumOpSetVersion, "OpSet version " + ctx.GetOpSetVersion() + " is older than " + LoaderSignature + "'s minimum OpSet version requirement: " + minimumOpSetVersion);
+            return _mapper is ICanSaveOnnx onnxMapper ? onnxMapper.CanSaveOnnx(ctx) : false;
```

Comment:
suggestion for exception message:
"Requested Opset version is lower than ...'s minimum Opset requirement"

Reply:
looks good